base-notebook/Dockerfile

```diff
@@ -95,17 +79,18 @@ RUN mkdir "/home/${NB_USER}/work" && \
 # Install conda as jovyan and check the sha256 sum provided on the download site
 WORKDIR /tmp
 
+# ---- Miniforge installer ----
+# Check https://github.com/conda-forge/miniforge/releases
+# Package Manager and Python implementation to use (https://github.com/conda-forge/miniforge)
+# - conda only: either Miniforge3 to use Python or Miniforge-pypy3 to use PyPy
+# - conda + mamba: either Mambaforge to use Python or Mambaforge-pypy3 to use PyPy
```

Comment:
You are outlining the miniforge release binaries, but we are using mambaforge at all time and that's it right? I got a bit confused about this listing as we have a hardcoded opinion about mambaforge atm, and this didn't mention it while listing various options.

Reply:
Concretely, I suggest just mentioning that we have opined to use the classic mambaforge installation.

Reply:
Yes, we're using mambaforge at all times, this is a small description of options available.

Reply:
Improved comment